spread/project.go

```diff
@@ -23,6 +23,8 @@ type Project struct {
 
 	Environment *Environment
 
+	Artifacts []string
```

Comment:
I think we should extend this to test suites, it would be nice to define artifact al suite level as well.